### PR TITLE
corss compiling fo acplt_build.tcl

### PR DESCRIPTION
**Pull request by LEvertz**
adapted the acplt_build.tcl script and come Makefiles in ov to allow for Cross compiling
Adapted ALL userlib Makefiles to use a cross option for acplt_makmak

### CODE REVIEW

**Comment by Stasik0** [comment]
This is top imho! I assume, it builds fine? Did you test mingw? Rpi-cross chain?

File: syslibs/comm/opcua/build/linux/Makefile [hunk 1]
```diff
@@ -17,14 +17,22 @@ COMPILER = GCC
...
+# Set TARGETOS to --targetWindows --targetLinux or --targetSolaris to cross-compile for the respective system.\n
+TARGETOS = 
+		
 all:
 
 %:
-	
 #	cd ../../open62541 && cmake -G "Eclipse CDT4 - Unix Makefiles" -DCMAKE_BUILD_TYPE=debug && $(MAKE) all && $(MAKE) open62541-objects
 #	ar rcs open62541.a ../../open62541/CMakeFiles/open62541-objects.dir/src/*.o ../../open62541/CMakeFiles/open62541-objects.dir/src/server/*.o ../../open62541/CMakeFiles/open62541-objects.dir/src_generated/*.o
+<<<<<<< HEAD
```
Comment (HolgerJeromin) [inline syslibs/comm/opcua/build/linux/Makefile]: Here is a merge problem :-)

**Comment by HolgerJeromin** [comment]
as you are at the core of compilation it would be nice, if you would overwrite/set the CC Macro only if it is not set. This would also be the requirement for #12 
BTW what is the purpose of $(PREFIX)?

**Comment by HolgerJeromin** [comment]
error output for travis should be working again after https://github.com/acplt/rte/commit/d09e4634388fa9eab33aa45fcd0ac7ae148965f6

**Comment by Stasik0** [comment]
@HolgerJeromin Perfix is needed for cross compilation

**Comment by LEvertz** [comment]
Problems in the Makefiles are fixed. Please review and merge.

**Comment by LEvertz** [comment]
I am on it again. Currently for RaspberryPi. The last changes were connected with that. Still working....

**Comment by LEvertz** [comment]
Currently the checks pass.
A line of commits for travis will follow here in order to release the builds.

**Pull request merged**